common/scripted_triggers/wc_portrait_triggers.txt

```diff
@@ -472,6 +472,10 @@ potential_race_stone_tolvir_trigger = {
 	culture = { has_graphical_culture = stone_tolvir_portrait_gfx }
 	# NOT = { has_trait = creature_stone_tolvir }
 }
+potential_race_highmountain_tauren_trigger = {
```

Comment:
Not placed in...
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114311917-bcd0b880-9b01-11eb-8a8a-1b0709b4ac7d.png)

</details>